crypto/crmf/crmf_local.h

```diff
@@ -76,7 +76,7 @@ struct ossl_crmf_encryptedkey_st {
     } value;
 } /* OSSL_CRMF_ENCRYPTEDKEY */;
 
-# if OPENSSL_VERSION_NUMBER <= 0x30200000L
+# if OPENSSL_VERSION_NUMBER <= 0x930200000L
```

Comment:
same as above.

Reply:
Yep, and several more such cases, all changed now.